### DISCUSSION TITLE
Hide voting fore selected user story

### DISCUSSION THREAD

**Discussion by drvic10k** [Zuehlke/poinz#341] [Ideas]
It would be nice to be able to select a story for estimation but hode the voting buttons until the discussion is done and team is ready to estimate, but at the same time, everybody sees the current ticket that is discussed

**Comment by xeronimus** [comment]
Hi @drvic10k ,
thanks for your Input. 

- How and who would "toggle" between hidden estimation cards and visible estimation cards?
- Why don't people "just wait" until the moderator says , it's ok to estimate?
- What is the harm, if people start voting during the discussion?   (no information is shared with others apart from "Jim already voted")

**Comment by drvic10k** [comment]
good points, I think we can manage by just giving prompts ourselves

I guess, when you get a tool, you want to use it for everything :)
